pitr/pitr.go

```diff
@@ -149,7 +154,7 @@ func createTiStore(urls string) (kv.Storage, error) {
 	return tiStore, nil
 }
 
-func getSnapshotMeta(tiStore kv.Storage) (*meta.Meta, error) {
+func GetSnapshotMeta(tiStore kv.Storage) (*meta.Meta, error) {
```

Comment:
We do not need this change ?

Reply:
DDL pitr test (in new package) wanna get this.